lib/opa/docker-compose.yml

```diff
@@ -21,13 +21,23 @@ services:
         target: opa-service-token
       - source: vault-approle-token
         target: vault-approle-token
+      - source: keycloak-client-local-candig-secret
+        target: /run/secrets/client_secret
+      - source: keycloak-test-user2-password
+        target: /run/secrets/password
     environment:
       KEYCLOAK_CLIENT_ID: ${KEYCLOAK_CLIENT_ID}
+      KEYCLOAK_REALM_URL: ${KEYCLOAK_REALM_URL}
+      KEYCLOAK_URL: ${KEYCLOAK_PRIVATE_URL}
       OPA_SITE_ADMIN_KEY: ${OPA_SITE_ADMIN_KEY}
       OPA_URL: ${OPA_PRIVATE_URL}
-      KEYCLOAK_REALM_URL: ${KEYCLOAK_REALM_URL}
       VAULT_URL: ${VAULT_PRIVATE_URL}
       SERVICE_NAME: ${SERVICE_NAME}
+    healthcheck:
+      test: [ "CMD", "python", "/app/healthcheck.py"]
```

Comment:
I think if there is a new script in opa you will need to commit it to that repo and make a PR there (If I am understanding correctly)

Reply:
Added! Here is the [link](https://github.com/CanDIG/candig-opa/pull/42)!